src/test/test-pretty-print.c

```diff
@@ -34,7 +34,7 @@ static void test_cat_files(void) {
 static void test_red_green_cross_check_mark(void) {
         bool b = false;
 
-        printf("yeah: <%s>\n", GREEN_CHECK_MARK());
+        printf("yea: <%s>\n", GREEN_CHECK_MARK());
```

Comment:
tsk, we should reject these Americanisms, and go with the much better Scots versions: `aye` or `nae` :-P

Reply:
Since the negative case says "nay" here, it should be "yay" here.

Reply:
I think it's actually spelled "yea". At least https://en.wiktionary.org/wiki/nay and other sources say so.

Reply:
(or "aye")